HadesScheduler/docker/docker.go

```
@@ -16,40 +15,42 @@ import (
+func streamContainerLogs(ctx context.Context, cli *client.Client, publisher buildlogs.LogPublisher, containerID, jobID string) error {
 	logReader, err := cli.ContainerLogs(ctx, containerID, client.ContainerLogsOptions{
 		ShowStdout: true,
 		ShowStderr: true,
 		Timestamps: true,
+		Follow:     true,
 	})
 	if err != nil {
-		return nil, nil, fmt.Errorf("getting container logs: %w", err)
+		return fmt.Errorf("getting container logs: %w", err)
 	}
 	defer logReader.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `logReader.Close` is not checked

(errcheck)